build-system/global-configs/canary-config.json

```diff
@@ -11,14 +11,11 @@
   "flexAdSlots": 0.05,
   "ios-fixed-no-transfer": 1,
   "ads-initialIntersection": 1,
-  "amp-cid-backup": 1,
   "flexible-bitrate": 0.1,
-  "disable-a4a-non-sd": 1,
   "story-ad-auto-advance": 1,
   "story-ad-placements": 1,
   "story-load-first-page-only": 1,
   "story-load-inactive-outside-viewport": 1,
   "story-disable-animations-first-page": 1,
-  "amp-story-page-attachment-ui-v2": 1,
```

Comment:
@gmajoulet 
@mszylkowski

Reply:
@processprocess is there anything we need to clean up from the page attachment experiment?

Reply:
(If it helps, the string `amp-story-page-attachment-ui-v2` is nowhere to be found.)